addons/web/tests/test_image.py

```diff
@@ -103,7 +103,8 @@ def test_04_web_content_filename_secure(self):
         })
 
         res = self.url_open(f'/web/image/{att.id}')
-        self.assertEqual(res.headers['Content-Disposition'], 'inline; filename="foo-l\'eb _ a\\"!r\\".gif"; filename*=UTF-8\'\'f%C3%B4%E2%98%BAo-l%27%C3%A9b%20_%20a%22%21r%22.gif')
+        expected_ufilename = url_quote(att.name.replace('\n', '_').replace('\r', '_'))
+        self.assertEqual(res.headers['Content-Disposition'], f'inline; filename="foo-l\'eb _ a\\"!r\\".gif"; filename*=UTF-8\'\'{expected_ufilename}')
```

Comment:
It's too hard to read :(
```suggestion
        self.assertEqual(res.headers['Content-Disposition'], r"""inline; filename="foo-l'eb _ a\"!r\".gif"; filename*=UTF-8''""" + expected_ufilename)
```